app/templates/frameworks/_framework_actions.html

```diff
@@ -25,7 +25,9 @@
 
 {% if framework.status == 'open' %}
 <li class="browse-list-item">
-  <a class="browse-list-item-link" href="{{ url_for('.framework_supplier_declaration', framework_slug=framework.slug, section_id=first_page_of_declaration) }}">
+  <a class="browse-list-item-link" href="{{ url_for('.framework_start_supplier_declaration', framework_slug=framework.slug)
```

Comment:
It's might be clearer overall to roll the conditional href and anchor text into one `declaration_status` check here? Something like 
```
{% if declaration_status == 'unstarted' %}
  <a href=your new page>Make supplier declaration</a>
(% else %}
  <a href=old link>Edit supplier declaration</a>
{% endif %}
```